### ISSUE TITLE
Dependency packages missing

### ISSUE THREAD

**Issue by Dark4Codrutz** [Ansuel/GUI_ipk#6]
I'd want to request for etherwake, it is needed for luci-app-wol, which is not present in this repository. Here is the link for the package from roleo's repo: https://repository.ilpuntotecnico.com/files/roleo/public/agtef/1.1.0/brcm63xx-tch/packages/packages/etherwake_1.09-3_brcm63xx-tch.ipk

On another similar topic, there seems to be a lot of dependant packages missing from this repository. Is it intended to be used besides roleo's repo ?
For eg there are these dependencies not met for luci when trying to upgrade to .302 version present in this repo:
 *      uhttpd *        luci-mod-admin-full *   libiwinfo-lua *         rpcd-mod-rrdns *

**Comment by Dark4Codrutz** [comment]
Am I missing something ? 
Are there issues with some packages, therefore not present in the repository ?
I'm asking because I remember there were some issues with uhttpd when luci was first getting ported... or maybe it wasn't

**Comment by nutterthanos** [comment]
You know @Ansuel has provided the toolchain so you could compile it yourself

**Comment by Dark4Codrutz** [comment]
> You know @Ansuel has provided the toolchain so you could compile it yourself

Yes, I've seen the commit posted the day after my issue was opened.

**Comment by nutterthanos** [comment]
oh
